app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -82,19 +86,20 @@ $(function() {
       var minimumInputLength = $(el).data('minimum_input_length');
       var order = $(el).data('order') || (fields[0] + '_desc');
       var parentId = $(el).data('parent_id') || INVALID_PARENT_ID;
+      var per_page = $(el).data('per_page');
```

Comment:
Unexpected var, use let or const instead                         no-var<br>Identifier 'per_page' is not in camel case                       camelcase<br>'$' is not defined                                               no-undef